.github/workflows/tests.yml

```diff
@@ -287,18 +378,28 @@ jobs:
             ${{ runner.os }}-minio-${{ env.CACHE_NUMBER }}
 
       - uses: conda-incubator/setup-miniconda@v2
+        name: Setup miniconda for defaults
+        if: matrix.default-channel == 'defaults'
+        env:
+          CONDA_SOLVER: classic
+        with:
+          activate-environment: conda-test-env
+
+      - uses: conda-incubator/setup-miniconda@v2
+        name: Setup miniconda for conda-forge
+        if: matrix.default-channel == 'conda-forge'
+        env:
+          CONDA_SOLVER: classic
         with:
           activate-environment: conda-test-env
-          miniconda-version: "latest"
-          python-version: ${{ matrix.python-version }}
-          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
```

Comment:
We don't need this anymore for caching?

Reply:
Without it we can't find newer packages (e.g. conda-build 3.23.3) on conda-forge, so it's not like we have a choice.